src/test/scala/edu/ie3/mobsim/model/TripSimulationSpec.scala

```diff
@@ -80,14 +79,14 @@ class TripSimulationSpec extends UnitSpec with ChargingBehaviorTestData {
           workPoi shouldBe givenWorkPoi
           storedEnergy shouldBe half
           chargingAtSimona shouldBe false
-          destinationPoiType shouldBe PoiTypeDictionary.CHARGING_HUB_TOWN
-          destinationCategoricalLocation shouldBe CategoricalLocationDictionary.CHARGING_HUB_TOWN
+          destinationPoi.getPoiType shouldBe PoiTypeDictionary.CHARGING_HUB_TOWN
+          destinationPoi.categoricalLocation shouldBe CategoricalLocationDictionary.CHARGING_HUB_TOWN
           destinationPoi shouldBe charging_hub_townPoi
           parkingTimeStart shouldBe simulationStart.plusMinutes(1)
           departureTime shouldBe simulationStart.plusHours(4).plusMinutes(33)
           chargingAtHomePossible shouldBe true
           chosenChargingStation shouldBe None
-          finalDestinationPoiType shouldBe None
+          finalDestinationPoi.map(_.getPoiType) shouldBe None
```

Comment:
Unnecessary mapping here, you could leave this as it was before

Reply:
The old way does not work anymore, because the field finalDestinationPoiType was removed from ``ElectricVehicle``.